docs/articles/licensing/index.rst

```diff
@@ -21,3 +21,5 @@ tooling in a future proof way. As SPDX is the licensing standard that
 compliance tooling will be using in the future, and indeed already is
 used by tool makers such as WindRiver and Black Duck, they are included
 both for support for tooling and future proofing the format.
+
+.. tags:: processes
```

Comment:
I think it should be singular "process" instead of processes. E.g. it's currently howto, not howtos and the tag makes more sense in singular form since it refers to the nature of one article.

Reply:
Fixed, changed the tag to "process".